backend/api/models/friend.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+import uuid
+
+# Related Model
+from api.models.author import Author
+
+
+# NOTE: When a friend record is created there should be two records made, with each user as the primary friend
+class Friend(models.Model):
+    # Unique Id of this friend record
+    friend_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False, unique=True)
```

Comment:
Could you explain a little on the usage of `friend_id` or friend record. I assume `author_id` of the friends is sufficient to identify friends

Reply:
friend id was not needed i agree, it was 2am haha. I am getting rid of it, i am keeping the author id and the friends id only